collections/_article/dev-snapshot-godot-4-2-beta-4.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Dev snapshot: Godot 4.2 beta 4"
+excerpt: "We continue iterating quickly on beta snapshots for Godot 4.2 to ensure that we can solve regressions before the stable release."
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/dev-snapshot-godot-4-2-beta-4.webp
+image_caption_title: Spooky FPS project
+image_caption_description: A game by Hannes Rahm
+date: 2023-10-31 12:00:00
+---
+
+We're well into the beta phase for Godot 4.2, with [4.2 beta 1](/article/dev-snapshot-godot-4-2-beta-1/) released 3 weeks ago, with new beta snapshots every week since. If you missed that release, have a look at the [release notes](/article/dev-snapshot-godot-4-2-beta-1/) for an overview of the key changes in Godot 4.2.
```

Comment:
```suggestion
We're well into the beta phase for Godot 4.2, with the first beta released 3 weeks ago, and new beta snapshots every week since. Have a look at the [4.2 beta 1 release notes](/article/dev-snapshot-godot-4-2-beta-1/) for an overview of the key changes in Godot 4.2.
```
Sounds better without two "with"s close together. I found the "that release" ambiguous, so rewrote to specify beta 1, and then rewrote the earlier text so it didn't say "4.2 beta 1" twice close together!